src/LAD4_1/lad4_1.cpp

```diff
@@ -117,10 +117,10 @@ void loop() {
     sensor = &sensors[j];
     sensorReadFunc(sensor->id);
     packet = make_packet(sensor->id, false);
-    Serial.println(packet);
+    Serial.print(packet);
 
     #if SERIAL_INPUT != 1
-        RFSerial.println(packet);
+        RFSerial.print(packet);
```

Comment:
Why are some of these print statements being changed from println() to print() and some arent?

Reply:
good catch. that's an accident